python/mxnet/metric.py

```diff
@@ -883,6 +983,17 @@ def get(self):
         """
         return (self.name, math.exp(self.sum_metric/self.num_inst))
 
+    def get_global(self):
+        """Returns the current global evaluation result.
+
+        Returns
+        -------
+        Tuple of (str, float)
+            Representing name of the metric and evaluation result.
+        """
+        num = self.global_num_inst if self.global_num_inst > 0 else float('nan')
```

Comment:
return float('nan')?

Reply:
Oh, this change got here by accident - I will revert it. Good catch, thank you!

Reply:
When I added tests, it turned out that changing this is actually necessary (I did change it slightly differently though, to match the `get` function from a base class) to avoid floating point division by 0 exception.